### PR TITLE
Update readme links

### PR DESCRIPTION
**Pull request by kusanele**

### CODE REVIEW

**Comment by pgleeson** [comment]
@kusanele This is a pull request to main, not development. You'll have to change it  (Edit button at the top). Be very careful when you open a pull request, examine it carefully, the red flag here was that 78 files were changed...

**Comment by kusanele** [comment]
I changed the directory of this pull request  to development.  however i am still getting the same error, not sure why.

**Comment by pgleeson** [comment]
@kusanele Have a look at this comment on formatting of code: https://github.com/ModECI/MDF/pull/358#issuecomment-1346831642

**Comment by kusanele** [comment]
Screenshot (30)

Good day @pgleeson , the dependences mentioned at [#358 (comment)](https://github.com/ModECI/MDF/pull/358#issuecomment-1346831642) Were installed as per the instructions given . however i am still getting the error message about the format.

Please fins the exact thing that is causing this error attached above. Thank you

**Comment by kusanele** [comment]
@pgleeson well , i have a new problem now. Git hub wont allow me to push the changes because its complaining about the file size.so i went on and followed some stack overflow solution that recommended that i remove the large files on my repo ,well that created more problems for me.

**Comment by kusanele** [comment]
Please kindly assist

File: docs/sphinx/source/api/Installation.md [hunk 1]
```diff
@@ -37,7 +37,9 @@ cd MDF
 ### 5) Install the package
 ```
 pip install .
+
 ```
+Hello world
```
Comment (sanjayankur31) [inline docs/sphinx/source/api/Installation.md:42]: I don't think we want this committed---it was just a test to see if your `make html` was working.

**Comment by sanjayankur31** [comment]
As we discussed in the meeting, it seems like your virtual environment folder was also added to the commit (`git log --stat` output). So, that's why github won't let you push---it's too big. You should only add and commit the readme file that you have modified. So, I think the simplest is to copy your readme file somewhere, create a new branch from development and paste it there and commit it to open a new pull request.

Let me know if you run into any issues---will be on slack most of the day :)

File: docs/sphinx/source/api/Installation.md (hunk 1 above)
Comment (pgleeson) [reply docs/sphinx/source/api/Installation.md:42]: @kusanele yes, this change shouldn't have been committed. Just change the file back locally and commit that.

**Comment by kusanele** [comment]
New pull request was created for the same purpose

File: docs/sphinx/source/api/Specification.rst [hunk 2]
```diff
@@ -201,3 +201,4 @@ Allowed field      Data Type             Description
 **receiver_port**  str                   The id of the InputPort on the receiver `Node <#node>`__
 **parameters**     Union[Any, NoneType]  Dictionary of parameters for the edge.
 =================  ====================  ============================================================================================================
+
```
Comment (pgleeson) [inline docs/sphinx/source/api/Specification.rst:204]: This change and the one above are probably not required and should be removed.

Run `pre-commit run --all-files` again on your local checkout of this branch and these should be reverted, and then commit again.

File: examples/MDF/README.md [hunk 3]
```diff
@@ -1,64 +1,81 @@
...
 
 ## Simple example
 
-[Python source](simple.py) | [JSON](Simple.json) | [YAML](Simple.yaml)
+[Python Source](https://github.com/ModECI/MDF/blob/update_readme_links/examples/MDF/simple.py) | [JSON](https://github.com/ModECI/MDF/blob/main/examples/MDF/Simple.json) | [YAML](https://github.com/ModECI/MDF/blob/main/examples/MDF/Simple.yaml)
 
 A simple example with 2 [Nodes](https://mdf.readthedocs.io/en/latest/api/Specification.html#node) connected by an [Edge](https://mdf.readthedocs.io/en/latest/api/Specification.html#edge):
 
-![simple](images/simple.png)
+<img src="https://raw.githubusercontent.com/ModECI/MDF/main/examples/MDF/images/simple.png" width="165" height="135" />
```
Comment (pgleeson) [inline examples/MDF/README.md:15]: Looks good in the generated html @kusanele, but double check how it looks in the markdown:
https://github.com/ModECI/MDF/blob/c3fda2e94b9585b407f29a3078530ab6f3925ae8/examples/MDF/README.md
Screenshot 2023-01-13 at 17 43 40

File: docs/sphinx/source/api/Specification.rst (hunk 2 above)
Comment (kusanele) [reply docs/sphinx/source/api/Specification.rst:204]: Please disregard this pull request. There were alot of problems with this branch hence i created a new branch

**Pull request closed without merging**